backend/de.metas.util/src/main/java/de/metas/util/Loggables.java

```diff
@@ -95,6 +94,20 @@ public static ILoggable withLogger(@NonNull final Logger logger, @NonNull final
 		return new LoggableWithLogger(get(), logger, level);
 	}
 
+	@NonNull
+	public static ILoggable withFallbackToLogger(@NonNull final Logger logger, @NonNull final Level level)
+	{
+		final ILoggable threadLocalLoggable = get();
+		if (NullLoggable.isNull(threadLocalLoggable))
+		{
+			return new LoggableWithLogger(threadLocalLoggable, logger, level);
```

Comment:
use NullLoggable.instance instead of threadLocalLoggable.
just to be on the same side (not sure if get() can return null or not)